package.json

```diff
@@ -30,9 +31,12 @@
     "typescript": "^3.2.2"
   },
   "dependencies": {
-    "debug": "^4.1.1",
+    "chalk": "^2.4.1",
```

Comment:
dependencies are run-time dependencies and are downloaded by the end-user of this library. The examples are not part of the end-user experience, any dependencies from examples should be put in `devDependencies`